### PR TITLE
Use ionicons in header

### PR DESCRIPTION
**Pull request by NotMyFault**
This is a follow-up to https://github.com/jenkinsci/jenkins/pull/7052/, which replaces left over icons in favor of ionicons, matching the overall icon scheme.

<details>
<summary>noJob old</summary>

url-old

</details>

<details>
<summary>noJob new</summary>

url-new

</details>

<details>
<summary>header/footer old</summary>

header-old

</details>

<details>
<summary>header/footer new</summary>

header-new

</details>

### Proposed changelog entries

* Replace header icons with ionicons.

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [ ] (If applicable) Jira issue is well described
- [ ] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change) and are in the imperative mood. [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  - Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadoc, as appropriate.
- [ ] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")` if applicable.
- [ ] New or substantially changed JavaScript is not defined inline and does not call `eval` to ease future introduction of Content-Security-Policy directives (see [documentation on jenkins.io](https://www.jenkins.io/doc/developer/security/csp/)).
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`:

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are accurate, human-readable, and in the imperative mood
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/7098"></a>

### CODE REVIEW

**Review by timja — APPROVED** [review]

**Comment by daniel-beck** [comment]
To me this replaces an icon that looks (at least somewhat) like "security" with a generic "warning" icon. I don't think this is an improvement.

**Comment by uhafner** [comment]
> To me this replaces an icon that looks (at least somewhat) like "security" with a generic "warning" icon. I don't think this is an improvement.

Maybe using the `shield` icon would be more appropriate (and an additional exclamation mark inside)?

**Comment by daniel-beck** [comment]
Could we build an icon like `shield-checkmark-outline` but with an exclamation point like `warning.svg` instead? I'd be happy to attempt it if license etc. are not a problem?

**Comment by daniel-beck** [comment]
> > To me this replaces an icon that looks (at least somewhat) like "security" with a generic "warning" icon. I don't think this is an improvement.
>

**Comment by NotMyFault** [comment]
> That would be great! I'd be happy to attempt it if license etc. are not a problem?

Yep, that works:

Screenshot 2022-09-15 at 10 46 34

File: war/src/main/resources/images/symbols/shield-warning.svg
Comment (daniel-beck) [inline war/src/main/resources/images/symbols/shield-warning.svg:1]: The exclamation point looks a little far down to me.

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" class="ionicon" viewBox="0 0 512 512"><title>Shield Warning</title><path d="M250.26 155.39l5.74 122 5.73-121.95a5.74 5.74 0 00-5.79-6h0a5.74 5.74 0 00-5.68 5.95z" fill="none" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="32"/><path fill="currentColor" d="M256 380.25a20 20 0 1120-20 20 20 0 01-20 20z"/><path d="M463.1 112.37C373.68 96.33 336.71 84.45 256 48c-80.71 36.45-117.68 48.33-207.1 64.37C32.7 369.13 240.58 457.79 256 464c15.42-6.21 223.3-94.87 207.1-351.63z" fill="none" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="32"/></svg>
```

WDYT? Before and after:

before
after

Comment (NotMyFault) [reply war/src/main/resources/images/symbols/shield-warning.svg]: I agree, looks better 👍🏻 
Thanks :)

**Review by daniel-beck — APPROVED** [review]

**Review by uhafner — APPROVED** [review]

**Comment by NotMyFault** [comment]
This PR is now ready for merge. We will merge it after ~24 hours if there is no negative feedback.
Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process.
Thanks!

**Pull request merged**
